_haikus/WhistleLoud.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: WhistleLoud
+author: Business
+---
+
+Can you Whistle Loud?<br>
+Striking Fear like an engine<br>
+Unaware of Quiet<br>
```

Comment:
This line is one syllable too long, that is why the test failed, so please don't override testing.

You can consider updating the line to this:
```
Quiet Unaware
```